libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts

```diff
@@ -102,12 +102,14 @@ export class AdminClientsService {
   async findByTenantIdAndClientId(
     tenantId: string,
     clientId: string,
+    checkArchived = false,
   ): Promise<AdminClientDto> {
     const client = await this.clientModel.findOne({
       where: {
         clientId,
         domainName: tenantId,
         enabled: true,
+        ...(checkArchived && { archived: null }),
```

Comment:
Shouldn't this be changed to the following instead

- When checkArchived true
```
{
  enabled: true,
}
```

- When checkArchived false:
```
{
  enabled: true,
  archived: null
}
```

Reply:
No I dont think so, the `enabled: true` was put there to fetch only those that have not been deleted, then we realised that `archived` was controlling if it had been deleted